docs/developers/omnichain/bitcoin.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Bitcoin: deposit & call contracts"
+sidebar_position: 7
+---
+
+# Deposit and call zEVM contracts from Bitcoin
+
+:::tip
+
+This section is important for wallet and app developers.
+
+:::
+
+To deposit BTC into zEVM ZRC-20 contract (and optionally call a smart contract),
+the Bitcoin transaction must conform to this specifications:
+
+1. The Bitcoin transaction must have at least 2 outputs;
+2. The first output must be addressed to the TSS Bitcoin address;
+3. The second output must be a memo output, i.e. `OP_RETURN PUSH_x [DATA]`. This
+   output must be less than 80 bytes;
+4. The memo `[DATA]` is an array of bytes that encodes the recipient address of
```

Comment:
perhaps we could have real examples of what `[DATA]` should look like for 4-6 here. i think it'd help people understand better what they need to use/encode for various use cases!

Reply:
Good idea! I think for users to also be able to test it out it should be part of `send-btc` task. Created an issue to add it once `send-btc` is a bit more user-friendly in terms of memo-handling.

https://github.com/zeta-chain/docs/issues/61